securedrop/source_templates/generate.html

```diff
@@ -17,17 +14,19 @@ <h2 id="codename-heading" class="visually-hidden">{{ gettext('Codename') }}</h2>
 </section>
 
 <p class="explanation">
-  {{ gettext('Because we do not track users of our <b>SecureDrop</b>
- service, in future visits, using this codename will be the only way we have to communicate with you should we have
- questions or are interested in additional information. Unlike passwords, there is no way to retrieve a lost codename.') }}
+  {{ gettext('You will need this codename to log into our SecureDrop later.') }}
+  <ol>
```

Comment:
UX tweak: apply consistent styling to text (my pref would be to use the styling on the list for the explanation as well, rather than vice versa, but either would be better).

Reply:
I've for now used the existing dark grey styling throughout, since it's used in other places as well, that will be overhauled as part of Michael's PR.